microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools-100.md

```diff
@@ -111,6 +111,35 @@ Now in Microsoft Edge 100, the selected theme is applied to the Waterfall view o
 For more information about using themes from Visual Studio Code in DevTools, see [Apply a color theme to DevTools](../../../customize/theme.md).  For more information about the Waterfall view for requests in the Network tool, see [Network features reference](../../../network/reference.md#display-the-timing-relationship-of-requests).
 
 
+<!-- ====================================================================== -->
+## Filter messages in the Issues tool by severity and browser target
```

Comment:
Can we move this section to after "The Network tool now displays how a request was fulfilled"? I believe that as long as we don't change the actual section titles, our in-product links will still work